### PR TITLE
only travis build against master

### PR DESCRIPTION
**Pull request by peterbe**
Now that we have dependabot making PRs, because of the way it works, they add the PR branches here into *this* repo. Not dependabot's own fork. 

What happens then is that Travis will build twice. Once for the PR ("Will this work if this PR is merged into master?") and once for the branch itself ("Will it work if this is the master branch?"). 
It's a wasted effort. 

We only need the PR build. 

Screen Shot 2019-08-16 at 9 33 57 AM

### CODE REVIEW

**Comment by peterbe** [comment]
As you can this, here in *this* there is only one Travis build. The one called "continous-integration/travis-ci/pr". That's because the branch lives on `peterbe`.

**Comment by peterbe** [comment]
By the way, this "trick" is often needed if you have a git repo in your own personal name and you make PRs against it. E.g. https://github.com/peterbe/hashin/pull/111
I've used [this trick](https://github.com/peterbe/hashin/blob/c3efcba050961257622c475114c087efe675c58a/.travis.yml#L8-L10) for years on projects where topic branches are uploaded to the main repo where the `master` branch exists.

**Comment by peterbe** [comment]
Also, https://github.com/mdn/stumptown-renderer/blob/2369a7c30de5ef802c50a6e268cecd169f3ea932/.travis.yml#L1-L4

**Comment by peterbe** [comment]
@ddbeck @wbamberg @Elchi3 
I've never really understood this but I asked for review from all 3 of you. That doesn't mean I need all 3 of you to approve. Just want *any* of you to review. 
However, what if the PR is really complex and requires multiple people's various skilled reviews?? 

Not sure what you guys prefer. 

Perhaps it's possible to create a *team* in the repo and then everybody in the team gets notified and *anybody* in the team can review. Lemme try that.

**Comment by peterbe** [comment]
Ok. I've created https://github.com/orgs/mdn/teams/core-stumptown-content/members and tried to invite you three gentlemen. Let's see if this works.

**Comment by ddbeck** [comment]
Do I understand correctly that we'll still have "Build pushed branches" and "Build pushed pull requests" turned on in Travis CI's web interface? In other words, it's true that we'll build for every PR (whether it's from a branch on this repo or from a fork) and we'll build for every push to this repo on `master`?

If so, then I'm happy with this change. I should probably do the same for BCD since we get the double builds (as described here and [in the Travis docs](https://docs.travis-ci.com/user/pull-requests/#double-builds-on-pull-requests)).

**Comment by peterbe** [comment]
Wat! It's [supposed to work](https://github.blog/2017-06-28-requesting-reviews-from-repository-teams/) but I can't seem to get the autocomplete to show up the team. Perhaps it's some caching.

**Comment by peterbe** [comment]
@ddbeck Yep you'll still get those. 1 build for every PR and 1 build for when it hits master.

**Comment by peterbe** [comment]
Ok I think I figured out the team thing. I had to add this repo to that team. As admin.

**Comment by peterbe** [comment]
So i'm curious what's going to happen now if any of you humans approve this. Will it know automatically that @mdn/core-stumptown-content approves?

**Comment by ddbeck** [comment]
> So i'm curious what's going to happen now if any of you humans approve this. Will it know automatically that @mdn/core-stumptown-content approves?

Let's find out! I'll try approving now.

**Review by ddbeck — APPROVED** [review]
🏗 🆗

**Comment by ddbeck** [comment]
>  ddbeck approved these changes on behalf of mdn/core-stumptown-content now 

🎉 Feel free to merge this, if my review is good enough. 😃

**Comment by peterbe** [comment]
Yay! Stuff works!
Screen Shot 2019-08-16 at 10 02 39 AM

Now, the big question is; What's the policy for merging? As in, who merges? The approver or the author?
I for one would be happy if the approver merges. 

(PS. I have worked in so many different teams and projects and repos that I no longer know whats up so I'll probably ask this question every couple of months)

**Comment by ddbeck** [comment]
> I for one would be happy if the approver merges.

IMO, if you request a review from `@mdn/core-stumptown-content` then the first approver should merge. If you request a review from named individuals, the _last_ approver should merge (i.e., if a reviewer wants unblock a merge, then they need to approve or dismiss themselves as a reviewer).

**Comment by peterbe** [comment]
So, @ddbeck will you merge this?

**Pull request merged**
